src/Microsoft.AspNetCore.Routing/Builder/DispatcherApplicationBuilderExtensions.cs

```diff
@@ -2,19 +2,39 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNetCore.Routing;
+using Microsoft.AspNetCore.Routing.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using System;
```

Comment:
sort

Reply:
Is it not already sorted?

Reply:
System should be first 😆  https://github.com/aspnet/Home/wiki/Engineering-guidelines#coding-style-guidelines--general

Reply:
oic

Reply:
I could make a job about the sort order being upside-down where you are. If it would put you at ease...

Reply:
![](https://i.imgur.com/fjwhd.gif)